Documentation/configuring-storage.md

```diff
@@ -34,6 +34,25 @@ If you left `spec.storage.hive.s3.createBucket` set to true, or unset, then you
 Please note that this must be done before installation.
 Changing these settings after installation will result in broken and unexpected behavior.
 
+## Storing data in Azure
+
+You can also store your data in Azure blob storage, to do so, you must use an existing container.
+Edit the `spec.storage` section in the example [azure-blob-storage.yaml][azure-blob-storage-config]] configuration.
+Set the  `spec.storage.hive.azure.container` and `spec.storage.hive.azure.storageAccountName` value.
```

Comment:
Let's instruct the users to only use an existing secret. The other stuff we don't want them to use directly, so we shouldn't mention setting `storageAccountName`, `secretAccessKey` or `createSecret`. Similar to S3.